internal/cmd/local/command_kafka.go

```diff
@@ -20,7 +21,7 @@ const (
 func (c *Command) newKafkaCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "kafka",
-		Short: "Manage a single-node instance of Apache Kafka.",
+		Short: "Manage a single-node or two-node instance of Apache Kafka.",
```

Comment:
The distinction between a single- and multi-node cluster should probably just be part of the description of `--multi-broker`
```suggestion
		Short: "Manage a local instance of Apache Kafka.",
```